_api-reference/document-apis/reindex.md

```diff
@@ -44,7 +44,7 @@ Parameter | Type | Description
 refresh | Boolean | If true, OpenSearch refreshes shards to make the reindex operation available to search results. Valid options are `true`, `false`, and `wait_for`, which tells OpenSearch to wait for a refresh before executing the operation. Default is `false`.
 timeout | Time | How long to wait for a response from the cluster. Default is `30s`.
 wait_for_active_shards | String | The number of active shards that must be available before OpenSearch processes the reindex request. Default is 1 (only the primary shard). Set to `all` or a positive integer. Values greater than 1 require replicas. For example, if you specify a value of 3, the index must have two replicas distributed across two additional nodes for the operation to succeed.
-wait_for_completion | Boolean | Waits for the matching tasks to complete. Default is `false`.
+wait_for_completion | Boolean | Setting this parameter to false indicates to OpenSearch it should not wait for completion and perform this request asynchronously. Asynchronous requests run in the background, and you can use the [Tasks]({{site.url}}{{site.baseurl}}/api-reference/tasks) API to monitor progress.
```

Comment:
```suggestion
wait_for_completion | Boolean |  When `false`, OpenSearch does not wait for the reindex to complete and instead performs this request asynchronously. Asynchronous requests run in the background. You can use the [Tasks]({{site.url}}{{site.baseurl}}/api-reference/tasks/) API to monitor progress.
```

Reply:
so your change is good.  However, it isn't clear that the default is "true", I thought the other wording maybe did that better.   Thoughts on how to make it clear the default for this parameter is true?

Reply:
```suggestion
wait_for_completion | Boolean |  When `false`, OpenSearch does not wait for the reindex to complete and instead performs this request asynchronously. Asynchronous requests run in the background. You can use the [Tasks]({{site.url}}{{site.baseurl}}/api-reference/tasks/) API to monitor progress. Default is `true`.
```

Reply:
I think you can tag the default to the end unless you want to add additional context for the default behavior.